rim/templates/rim/add.html

```diff
@@ -0,0 +1,37 @@
+{% extends "rim/base.html" %}
+{% load static %}
+{% load i18n %}
+<!DOCTYPE html>
+<html>
+    <head>
+        {% block css %}
+            {{ block.super }}
+            <link rel='stylesheet' type='text/css' href='{% static "rim/css/add.css" %}'>
+        {% endblock %}
+    </head>
+
+    {% block content %}
+        <body>
+            <div class='addtitle'>Add an item</div>
+            <div class='addform'>
+                <form method=POST class='addform'> {% csrf_token %}
+                    <div class="grid-x">
+                        {% for field in form %}
+                            <div class='small-4 large-4 addcell cell'>
+                                {{ field.help_text }}{{ field }}
```

Comment:
You should find a place to display [`{{ field.errors }}` and `{{ form.non_field_errors }}`](https://docs.djangoproject.com/en/2.0/topics/forms/#rendering-form-error-messages).